platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,480 @@
+---
+title:  "Platform Engineering Maturity Model"
+pdf: https://github.com/cncf/tag-app-delivery/raw/main/platforms-maturity-model/v1/assets/platform-eng-maturity-model-v1.0.pdf
+version_info: https://github.com/cncf/tag-app-delivery/tree/main/platforms-maturity-model/README.md
+description: "This maturity model intends to provide tactical guidance to users seeking to adopt the patterns discussed in the Platforms Definition White Paper. That paper suggests why and what to build; this document will begin to describe how to plan to build it. The target audience is CTOs, Directors of engineering, and lead engineers and architects seeking to evaluate their current state and environment and identify opportunities for improvement.
+<br/>
+<br/>
+This document refers to, enhances, and follows similar standards as the following related documents:
+<br/>
+[Cloud Maturity Model](https://maturitymodel.cncf.io/)
+<br/>
+[Platforms Definition White Paper](https://tag-app-delivery.cncf.io/whitepapers/platforms/)"
+type: whitepapers
+url: /whitepapers/platform-eng-maturity-model
+---
+
+## Introduction
+
+Inspired by the cross-functional cooperation promised by DevOps, platforms and platform engineering have emerged in enterprises as an explicit form of that cooperation. Platforms curate and present common infrastructure capabilities, frameworks, and experiences that facilitate and accelerate the work of internal users such as product and application teams.
+
+CNCF's initial "Platforms Definition" white paper describes what internal platforms for cloud computing are and the values they promise to deliver to enterprises. But to achieve those values an organization must reflect and deliberately pursue outcomes and practices that are impactful for them, keeping in mind that every platform is an internal platform crafted for its own organization. This maturity model provides a framework for that reflection and for identifying opportunities for improvement in any organization.
```

Comment:
Should we link it to the initial whitepaper here instead of linking it later in the document? https://tag-app-delivery.cncf.io/whitepapers/platforms/